locale/fi/LC_MESSAGES/django.po

```diff
@@ -1049,6 +1052,9 @@ msgstr "Varauksia enintään per käyttäjä (voimassa olevia)"
 msgid "Reservable"
 msgstr "Varattavissa"
 
+msgid "Should be reserved for whole day"
+msgstr "Vain kokopäivän varauksia"
```

Comment:
IMO this should be "Vain koko päivän varauksia", but if someone else has told you otherwise then he/she is probably right 🙂